docs/index.rst

```diff
@@ -0,0 +1,20 @@
+Welcome to toolchest's documentation!
+======================================
+
+.. toctree::
+   :maxdepth: 2
+   :caption: Contents:
+
+   readme
+   installation
+   usage
+   modules
```

Comment:
For atkinson and git_wrapper using source/modules seems to keep the code in the docs up to date.